app/build.gradle

```diff
@@ -143,6 +143,9 @@ dependencies {
     implementation "org.mozilla.components:feature-session:$moz_components_version"
```

Comment:
(unrelated to this LOC)

I wonder if we should rename `SessionRepo` -> `SelectedSessionRepo`, given these changes?